### PR TITLE
Update Contribution Guide

### PR DESCRIPTION
**Pull request by circleen**
Add a warning saying that you put your changes in a new branch.

### CODE REVIEW

File: wiki/osu!_wiki_Contribution_Guide/GitHub_Web_Interface/en.md [hunk 1]
```diff
@@ -80,7 +80,7 @@
 
 ![](img/online-commit-changes-empty.jpg "Please don't leave these empty!")
 
-2\. **Whatever you do, never leave the title textbox empty!** Instead, enter the article's title and a short summary of all of the changes you made **in English**.
+2\. **Whatever you do, never ever leave the title textbox empty!** Instead, enter the article's title and a short summary of all of the changes you made **in English**.
```
Comment (cl8n) [inline wiki/osu!_wiki_Contribution_Guide/GitHub_Web_Interface/en.md]: why

File: wiki/osu!_wiki_Contribution_Guide/GitHub_Web_Interface/en.md [hunk 2]
```diff
@@ -1,6 +1,6 @@
 # GitHub Web Interface
 
-*This article continues from the main page.* If you are going to make changes to multiple articles (this includes uploading, deleting, and/or moving images or files), please see the other guide, [GitHub Desktop](/wiki/owcg/GitHub_Desktop).
+*This article continues from the main page.* If you are going to make changes to multiple articles (this includes uploading, deleting, and/or moving images or files), please see the other guide [GitHub Desktop](/wiki/owcg/GitHub_Desktop).
```
Comment (cl8n) [inline wiki/osu!_wiki_Contribution_Guide/GitHub_Web_Interface/en.md]: comma is necessary if sentence is kept as-is

File: wiki/osu!_wiki_Contribution_Guide/GitHub_Desktop/en.md [hunk 3]
```diff
@@ -40,7 +40,7 @@
 
 ## Editing
 
-1\. In the folder you cloned the repo into, navigate to the file you want to edit. The articles are organized by the folder names being the article names and the `.md` files being the locales.
+1\. In the folder you cloned the repo into, navigate to the file you want to edit. The articles are organized by the folder names being the article names and the `.md` files being the translations.
```
Comment (cl8n) [inline wiki/osu!_wiki_Contribution_Guide/GitHub_Desktop/en.md]: better as-is (`en.md` isn't a translation)

File: wiki/osu!_wiki_Contribution_Guide/GitHub_Web_Interface/en.md (hunk 1 above)
Comment (circleen) [reply wiki/osu!_wiki_Contribution_Guide/GitHub_Web_Interface/en.md]: idk. just sounded more striking

**Comment by circleen** [comment]
Ready for review.

File: wiki/osu!_wiki_Contribution_Guide/en.md [hunk 4]
```diff
@@ -52,11 +52,13 @@ If, during any part, you are confused and/or need further help, feel free to sen
 
 ## Editing online or locally
 
-At this point, you have two choices:
+Here, you have two choices:
 
 - [GitHub Web Interface](/wiki/owcg/GitHub_Web_Interface) - edit online; this is best for single article edits
 - [GitHub Desktop](/wiki/owcg/GitHub_Desktop) - edit locally; this is best for single and/or multi article edits (this includes uploading, deleting, and moving images or files)
 
+**Also, be sure to put your local changes in a new branch. Putting your changes in the master branch of your fork and making a pull request with it is an anti-PR.**
```
Comment (MegaApplePi) [inline wiki/osu!_wiki_Contribution_Guide/en.md]:
```suggestion
The general gist is to use meaningful commit messages and use branches to keep your changes separate from your master branch. A new branch from master should be used when making changes to different articles or when making a new one.
```

**Comment by circleen** [comment]
Closing because I had to delete fork and 'unknown repository'

**Pull request closed without merging**